src/app/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import defaultAvatar from './defaultAvatar.svg';
+import classes from './Profile.module.css';
+
+type Profile = {
+  username: string | null;
+  displayName: string | null;
+  avatar: string | null;
+};
+
+function openDialog() {
+  overwolf.profile.openLoginDialog();
+}
+
+function Profile(): JSX.Element {
+  const defaultProfile: Profile = {
+    username: 'Game-Ticker',
+    displayName: 'Game-Ticker',
+    avatar: null,
+  };
```

Comment:
You can move this to prevent recreating this object on each render.

```suggestion
const defaultProfile: Profile = {
  username: 'Game-Ticker',
  displayName: 'Game-Ticker',
  avatar: null,
};
  
function Profile(): JSX.Element {
```

Reply:
jap thanks, solved